astroquery/esa/hubble/core.py

```diff
@@ -215,6 +219,85 @@ def get_observation_type(self, observation_id):
             raise ValueError("Invalid Observation ID")
         return obs_type
 
+    def get_observations_from_program(self, program, *, output_file=None,
```

Comment:
I wonder whether there is indeed any need for this convenience method at the price of complicating the API, as all it does is to pass on the parameters to `query_criteria` (maybe a well written example in the docs would be be sufficient)

Reply:
Thanks for the explanation, it sounds good.